### ISSUE TITLE
Document how data may be cached/stored

### ISSUE THREAD

**Issue by allasm** [MatchMakerExchange/mme-apis#42]
The current agreement is that data should not be stored but may be (temporarily?) cached, and that once a new response is obtained the entire cache should be replaced (correct?)

**Comment by Relequestual** [comment]
I don't know if we agreed on caching or not. I think we agreed it should not be stored in the same way that the receiving databases own data is stored. The intention is to make it clear that one system should not asimilate any recieved data into their dataset / database.

I know we DID agree that we could store the data for the purposes of logging and auditing. Even if our applications aren't doing that, it is probably being done at some firewall / higher network level anyway already.

**Comment by fschiettecatte** [comment]
Agree with @Relequestual

**Comment by buske** [comment]
I agree. I also think this is outside of the scope of the MME API itself, and is instead part of the policies adopted by the MME that govern it's use of the API. I think the API should be as agnostic about it's own use as possible.

**Comment by Relequestual** [comment]
I think the current documentation found at http://www.matchmakerexchange.org/assets/files/MatchmakerExchangeServiceRequirements.pdf item 7 covers this issue. @allasm ?
